Dockerfile

```diff
@@ -20,3 +20,8 @@ RUN find . -iname '*.o' -exec rm {} \; && \
     find . -iname '*.a' -exec rm {} \;
 
 COPY . /app/
+
+# This fixes "server is already running" issue.
+# https://stackoverflow.com/questions/35022428/rails-server-is-still-running-in-a-new-opened-docker-container
+ENTRYPOINT ["scripts/docker-entry.sh"]
```

Comment:
Looking good!

Should the "rails server" command below go into the entry script? Depends on mostly I think if you think the user should be able to run the container without starting the rails server.
https://www.ctl.io/developers/blog/post/dockerfile-entrypoint-vs-cmd/

Reply:
Yes, I do want users to be able to run other commands. Rails server runs the webapp, but there's a whole bunch of other commands that need to be run such as `rails console` or `rails db:migrate`. 

```
± |fix_tmp_pids ✓| → docker-compose run --rm web rails c
Starting dev_school_db_1 ... done
Loading development environment (Rails 5.2.1)
irb(main):001:0> ActiveRecord::Base.connected?
=> false
```

Reply:
(`rails c` == `rails console` btw :) )